core/src/main/java/org/apache/brooklyn/enricher/stock/Enrichers.java

```diff
@@ -891,6 +892,39 @@ protected static int count(Iterable<? extends Object> vals, boolean includeNullV
         return result;
     }
     
+    @Beta
+    public static class ComputingIsQuorate<T extends Object> implements Function<Collection<T>, T> {
```

Comment:
I can't escape the generics either. However, I can cast `ComputingIsQuorate` to `(Function<? super Collection<?>, ?>)` in Aggregator! Personally I prefer keeping ComputingIsQuorate simple and the ugliness in `lookupTransformation`.